src/xunit.console.netcore/Visitors/StandardOutputVisitor.cs

```diff
@@ -35,6 +47,11 @@ protected override bool Visit(ITestAssemblyStarting assemblyStarting)
             lock (consoleLock)
                 Console.WriteLine("Starting:    {0}", Path.GetFileNameWithoutExtension(assemblyFileName));
 
+            clock.Start();
+            watcher = new Thread(new ThreadStart(testWatcher));
```

Comment:
The "modern" way would probably be to use a Task, with TaskCreationOptions.LongRunning so it didn't use the thread pool. But this is fine.